cli/services/package_service/package_service.go

```
@@ -48,24 +60,29 @@ func (packageService PackageService) GetAll() []models.Package {
 func (packageService PackageService) InstallPackage(pkg models.Package, bbeConfig models.BbeConfig) error {
 	for _, p := range packages {
 		if p.Package.Name == pkg.Name {
-			cmd := execCommand("helm", "repo", "add", p.PackageRepository.Name, p.PackageRepository.RepositoryUrl)
-			if err := cmd.Run(); err != nil {
-				return fmt.Errorf("failed to add helm repository %s: %w", p.PackageRepository.Name, err)
-			}
+			if !IsPackageInstalled(pkg) {
+				cmd := execCommand("helm", "repo", "add", p.PackageRepository.Name, p.PackageRepository.RepositoryUrl)
+				if err := cmd.Run(); err != nil {
+					return fmt.Errorf("failed to add helm repository %s: %w", p.PackageRepository.Name, err)
+				}
 
-			cmd = execCommand("helm", "install", pkg.Name, fmt.Sprintf("%s/%s", p.PackageRepository.Name, p.HelmChart),
-				"--version", p.HelmChartVersion,
-				"--namespace", pkg.Name,
-				"--create-namespace",
-				"--kube-context", bbeConfig.Bbe.Cluster.Context)
-			if err := cmd.Run(); err != nil {
-				return fmt.Errorf("failed to install helm package %s: %w", pkg.Name, err)
+				cmd = execCommand("helm", "install", pkg.Name, fmt.Sprintf("%s/%s", p.PackageRepository.Name, p.HelmChart),
+					"--version", p.HelmChartVersion,
+					"--namespace", pkg.Name,
+					"--create-namespace",
+					"--kube-context", bbeConfig.Bbe.Cluster.Context)
+				if err := cmd.Run(); err != nil {
+					return fmt.Errorf("failed to install helm package %s: %w", pkg.Name, err)
+				}
+				logger.Info(fmt.Sprintf("Installed package %s", pkg.Name))
+				return nil
 			}
-			logger.Info(fmt.Sprintf("Installed package %s", pkg.Name))
+
 			return nil
 		}
+		return fmt.Errorf("Package `%s` not found", pkg.Name)
 	}
-	return fmt.Errorf("package %s not found", pkg.Name)
+	return nil
```

Surely we should return an error if you ask to install a package that does not exist, as we did before?